client/containers/product-variant/productsContainerCustomer.js

```diff
@@ -48,18 +44,6 @@ function composer(props, onData) {
     queryParams.query = queryString.query;
   }
 
-  // BOF: swag shop featuredProduct filter
```

Comment:
@zenweasel I removed these few unused lines which were originally intended to limit the featured products on the homepage to 3. Instead, I linked the "Shop swag" button on the homepage to the "Products we love" section, which currently displays all published products. The "Shop swag" button previously did nothing. Let me know if this is a good user experience, or if we should instead create a "Shop" tag, have all products assigned to it, and link the hero button to that tag page.